Content.Client/Doors/FirelockSystem.cs

```diff
@@ -26,11 +26,7 @@ private void OnAppearanceChange(EntityUid uid, FirelockComponent comp, ref Appea
             state = DoorState.Closed;
 
         boltedVisible = _appearanceSystem.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component) && lights;
-        unlitVisible =
-            state == DoorState.Closing
-            ||  state == DoorState.Opening
-            ||  state == DoorState.Denying
-            || (_appearanceSystem.TryGetData<bool>(uid, DoorVisuals.ClosedLights, out var closedLights, args.Component) && closedLights);
+        unlitVisible = comp.IsLocked;
```

Comment:
I'm actually mirroring old behavior here. Perhaps this is out of scope for this PR. I also think the behavior now is better, because it is always clear what the intended state of the airlock is, whether it's being pried open or not.

Reply:
Whatever the new or old behavior is, is fine. I just want to check that with the always lit functionality, that the lights are properly set to the correct lights in the cases where they were not on before.

Opening/closing the door always sets lights to green. If the door is open or closed the lights should be off or when IsLocked they should be orange.

Reply:
https://github.com/space-wizards/space-station-14/assets/10494922/69fdc8cd-393d-431b-9db8-0774915d2ba5

In SS13, the side lights turn green when opening/closing. Off when closed/open.

In SS14, we had green when opening/closing. Orange when IsLocked and open/closed. Off when not IsLocked and open/closed.